pkg/apis/smith/v1/doc.go

```diff
@@ -1,5 +1,3 @@
-// +k8s:deepcopy-gen=package,register
```

Comment:
@nilebox note how I removed package-wide deep copy generation and registration and instead marked specific types with `+k8s:deepcopy-gen=true`. That way I don't get the ugly `init()` and don't get the unnecessary deep copy methods on everything.

Reply:
Details here https://github.com/kubernetes/code-generator/blob/master/cmd/deepcopy-gen/main.go

Reply:
I mean, we should do the same in Service Catalog.

Reply:
Hm, in this case I actually don't see any difference (apart from omitted `init()`)...